### PR TITLE
D8Core-7476: Site Type

### PR DESCRIPTION
**Pull request by buttonwillowsix**
# READY FOR REVIEW

# Summary
- New field for tracking site type as per agreed upon categories for reporting with SODA

# Review By (Date)
- Today?

# Criticality
- Handy, but not on fire

# Urgency
- Medium

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Go to Site Settings
3. Categorize the site at the bottom of the Site Ownership page

### Site Configuration Sync

- YES

## Front End Validation
No front-end impact

### Code
- [YES ] Are the naming conventions following our standards?
- [ YES] Does the code have sufficient inline comments?
- [ NOPE] Is there anything in this code that would be hidden or hard to discover through the UI?
-[ NOPE] IAre there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ NOPE] I Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
-[ NOPE] I Is there anything included in this PR that is not related to the problem it is trying to solve?
-[ NOPE] IIs the approach to the problem appropriate?

# Affected Projects or Products
- CORE

# Associated Issues and/or People

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: field.field.config_pages.stanford_basic_site_settings.su_site_type.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+uuid: 18918dc6-5dca-49ee-be52-a2d0c57ac172
```
Comment (pookmish) [inline field.field.config_pages.stanford_basic_site_settings.su_site_type.yml]: This file should be in the `config/sync` directory

File: field.storage.config_pages.su_site_type.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+uuid: aef7044a-cf89-41f3-b683-aff011270c5b
```
Comment (pookmish) [inline field.storage.config_pages.su_site_type.yml]: This file should be in the `config/sync` directory

**Review by pookmish — APPROVED** [review]
Those two file should be moved, but it's probably good otherwise.

File: field.storage.config_pages.su_site_type.yml (hunk 2 above)
Comment (buttonwillowsix) [reply field.storage.config_pages.su_site_type.yml]: *facepalm* the two files are moved.

File: field.field.config_pages.stanford_basic_site_settings.su_site_type.yml (hunk 1 above)
Comment (buttonwillowsix) [reply field.field.config_pages.stanford_basic_site_settings.su_site_type.yml]: Fixed!

**Review by pookmish — APPROVED** [review]

**Pull request merged**